diff --git a/src/command-line/release/utils/git.js b/src/command-line/release/utils/git.js
index 383d50fe3d68f0ce36a97bcb39287416ee82911d..ebb87c3daf2840b1235fc828f087cc0946e8c148 100644
--- a/src/command-line/release/utils/git.js
+++ b/src/command-line/release/utils/git.js
@@ -27,6 +27,9 @@ async function getLatestGitTagForPattern(releaseTagPattern, additionalInterpolat
     try {
         let tags;
         tags = await (0, exec_command_1.execCommand)('git', [
+            // Apply git config to take version suffixes into account when sorting, e.g. 1.0.0 is newer than 1.0.0-beta.1
+            '-c',
+            'versionsort.suffix=-',
             'tag',
             '--sort',
             '-v:refname',
@@ -38,7 +41,14 @@ async function getLatestGitTagForPattern(releaseTagPattern, additionalInterpolat
             .filter(Boolean));
         if (!tags.length) {
             // try again, but include all tags on the repo instead of just --merged ones
-            tags = await (0, exec_command_1.execCommand)('git', ['tag', '--sort', '-v:refname']).then((r) => r
+            tags = await (0, exec_command_1.execCommand)('git', [
+                // Apply git config to take version suffixes into account when sorting, e.g. 1.0.0 is newer than 1.0.0-beta.1
+                '-c',
+                'versionsort.suffix=-',
+                'tag',
+                '--sort',
+                '-v:refname'
+            ]).then((r) => r
                 .trim()
                 .split('\n')
                 .map((t) => t.trim())
